### ISSUE TITLE
Latest release (3.7.0) broke our cocoapods usage

### ISSUE THREAD

**Issue by dtweston** [bitstadium/HockeySDK-iOS#178]
We reference HockeySDK as a dependency from a podspec, (i.e. not directly in a Podfile), and the latest release has broken this usage.

We have an app A with a Podfile that references our custom pod B that has a dependency on HockeySDK.

The xcconfig for our Pod B is missing the FRAMEWORK_SEARCH_PATHS setting that points to the HockeySDK framework location, but the setting shows up in the main app target. So, the classes that reference HockeySDK in Pod B fail to compile because they can't find the header. Is there any way that this can be resolved in the podspec?

For now, we've reverted back to v3.6.4

**Comment by lumaxis** [comment]
I'm not entirely sure if I understand your scenario correctly. Basically what you're saying is that our podspec doesn't work as a dependency in another podspec?
Do you think it would be possible to provide an example project that demonstrates the problem? It might be difficult to reproduce your scenario otherwise. Thanks!

**Comment by lukasspiess** [comment]
I'm not entirely sure if I understand your scenario correctly. Basically what you're saying is that our podspec doesn't work as a dependency in another podspec?
Do you think it would be possible to provide an example project that demonstrates the problem? It might be difficult to reproduce your scenario otherwise. Thanks!

**Comment by dtweston** [comment]
Sure, I think I can whip up an example project.

**Comment by anlinde** [comment]
Also try to completely remove it first and then add it again. Setting the build settings at the correct places can't be influenced by the pod spec anyway. So if deleting and adding doesn't help a test project might be the only chance to figure out what is going on.

**Comment by dtweston** [comment]
Here's my example project:
https://github.com/dtweston/hockeysdk-test

There are two relevant commits. One with 3.6.4 works, the next one with 3.7.0 fails

**Comment by Spencevail** [comment]
Same issue here. Here's my podfile:

```ruby
platform :ios, '8.0'

use_frameworks!

pod 'HockeySDK',                        '~> 3.x'
pod 'ReactKit',                         '~> 0.x'
pod 'Argo',                             '~> 1.x'
pod 'MagicalRecord',                    '~> 2.x'

target 'InspiratoTests' do
    pod 'Nimble',                       '~> 0.x'
    pod 'OHHTTPStubs',                  '~> 4.x'
end
```

The error I'm seeing when building is:

> /Users/Spencevail/Library/Developer/Xcode/DerivedData/MyApp-ecqengvbpfcaktaslcjliyhxlqsr/Build/Products/Staging-Debug-iphonesimulator/HockeySDKResources.bundle
building file list ... rsync: link_stat "/Users/Spencevail/Library/Developer/Xcode/DerivedData/MyApp-ecqengvbpfcaktaslcjliyhxlqsr/Build/Products/Staging-Debug-iphonesimulator/HockeySDKResources.bundle" failed: No such file or directory (2)
done

> sent 29 bytes  received 20 bytes  98.00 bytes/sec
total size is 0  speedup is 0.00
rsync error: some files could not be transferred (code 23) at /SourceCache/rsync/rsync-45/rsync/main.c(992) [sender=2.6.9]
Command /bin/sh failed with exit code 23

**Comment by anlinde** [comment]
@dtweston: Please use the `HockeySDK-Source` podspec, which is essentially the same as we had for 3.6.4 with updated source.

@Spencevail: We just released a new version 3.7.1 with updated podspec that should work for you know. Sorry for the trouble.

**Comment by Spencevail** [comment]
@anlinde I'm getting the same issue with 3.7.1. I even tried removing the Podfile.lock, the Pod directory, removing all derived data, reinstalling and rebuilding. Same error as before. Downgrading to 3.6.4 fixes the problem.

**Comment by anlinde** [comment]
We get reports that it works fine. Feel free to use 'Hockey-Source' pod instead which uses the same approach as 3.6.4

**Comment by sjmadsen** [comment]
We ran into this same problem with CocoaPods 0.37.2, but it is fixed in 0.38.0.beta.1. (See issue [#3761](https://github.com/CocoaPods/CocoaPods/issues/3761).) It may only be a problem when `use_frameworks!` is used (which is required if you use pure Swift pods).
